doc/changes/latest.inc

```diff
@@ -31,4 +31,4 @@ Bugs
 
 API changes
 ~~~~~~~~~~~
-- None yet
+- Add the argument ``max_iter`` to :func:`~mne.time_frequency.psd_array_multitaper` and to :func:`~mne.time_frequency.csd_array_multitaper` to increase the maximum number of iterations to reach convergence when using ``adaptive=True`` (:gh:`11237` by `Mathieu Scheltienne`_)
```

Comment:
`API` is really meant for backward incompatible changes. Since this just adds a new parameter that people can tweak, it should go under the `Enhancements` section.

Reply:
I did hesitate for a solid 2s 😅

Reply:
Yes our section names are a bit ambiguous...